client/components/AddToPlanningModal/index.jsx

```diff
@@ -8,12 +8,31 @@ import {AddToPlanningApp} from '../../apps';
 
 import * as selectors from '../../selectors';
 import {gettext} from '../../utils';
+import {KEYCODES} from '../../constants';
 
 export class AddToPlanningComponent extends React.Component {
     constructor(props) {
         super(props);
 
         this.dom = {popupContainer: null};
+        this.handleKeydown = this.handleKeydown.bind(this);
+        this.handleHide = this.props.handleHide;
```

Comment:
Whenever you're accessing anything from `props`, it is always better to use `this.props` everywhere in this file.
This makes it clear that you're accessing something passed down from a higher up component